lib/std/Thread/Condition.zig

```diff
@@ -41,6 +45,13 @@ pub const SingleThreadedCondition = struct {
         unreachable; // deadlock detected
     }
 
+    pub fn timedWait(cond: *SingleThreadedCondition, mutex: *Mutex, timeout_ns: u64) error{TimedOut}!void {
+        _ = cond;
+        _ = mutex;
+        _ = timeout_ns;
+        unreachable; // deadlock detected
```

Comment:
A timed wait on single threaded platforms doesn't necessarily dictate a deadlock. 
Could this be something like `std.time.sleep(timeout_ns); return error.TimedOut` instead?